src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -537,6 +539,7 @@ export default class DefaultAudioVideoController implements AudioVideoController
 
       this.sessionStateController.perform(SessionStateControllerAction.FinishConnecting, () => {
         this.actionFinishConnecting();
+        this.meetingSessionContext.pingPong.start();
```

Comment:
Why is this not in actionFinishConnecting?

Reply:
can be moved to be a part of the actionFinishConnecting()